packages/terra-i18n/src/intl.js

```diff
@@ -0,0 +1,4 @@
+import IntlPolyfill from 'intl';
+
+const exportedIntl = typeof (Intl) === 'undefined' ? IntlPolyfill : global.Intl;
```

Comment:
@tbiethman you and I have been working on removing the global XFC var in apps. Do you think a global is desired here?

Reply:
`global` could be replaced with `window`.

Reply:
This is exporting the global Intl that already exists in some browsers, not making a new global as part of the terra-i18n consumer API. The problem with the XFC implementations we're seeing are the global values that the various wrappers are making for direct consumer usage.

Reply:
Update to `global` to `window`: https://github.com/cerner/terra-core/pull/2288/commits/6d9d55dc9ed7cf2fdaac9134e2be73841d26163d